google-cloud-examples/src/main/java/com/google/cloud/examples/spanner/snippets/DatabaseClientSnippets.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * EDITING INSTRUCTIONS
+ * This file is referenced in DatabaseClient's javadoc. Any change to this file should be reflected
+ * in DatabaseClient's javadoc.
+ */
+
+package com.google.cloud.examples.spanner.snippets;
+
+import com.google.cloud.Timestamp;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.Key;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.ReadOnlyTransaction;
+import com.google.cloud.spanner.Struct;
+import com.google.cloud.spanner.TimestampBound;
+import com.google.cloud.spanner.TransactionContext;
+import com.google.cloud.spanner.TransactionRunner;
+import com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class contains snippets for {@link DatabaseClient} interface.
+ */
+public class DatabaseClientSnippets {
+
+  private final DatabaseClient dbClient;
+
+  public DatabaseClientSnippets(DatabaseClient dbClient) {
+    this.dbClient = dbClient;
+  }
+
+  /**
+   * Example of blind write.
+   */
+  // [TARGET write(Iterable)]
+  // [VARIABLE my_singer_id]
+  public void write(long singerId) {
+    // [START write]
+    List<Mutation> mutations = new ArrayList<>();
```

Comment:
This is a size 1 (`Collections.singletonList()`) list. What is the reason to use `ArrayList` here (as well as in other places, like line 79)?